src/common/hf.ts

```diff
@@ -14,7 +14,13 @@ import { existsSync, mkdirSync, writeFileSync } from 'fs'
 import { applyServerSide } from './k8s'
 
 const replaceHFPaths = (output: string, envDir = env.ENV_DIR): string => output.replaceAll('../env', envDir)
-export const HF_DEFAULT_SYNC_ARGS = ['sync', '--concurrency=1', '--sync-args', '--disable-openapi-validation --qps=20']
+export const HF_DEFAULT_SYNC_ARGS = [
+  'sync',
+  '--concurrency=1',
+  '--reuse-values', // Preserve values from existing releases on retry - makes install idempotent
+  '--sync-args',
+  '--disable-openapi-validation --qps=20',
```

Comment:
Was this way before, but better would be
```suggestion
  '--disable-openapi-validation',
  '--qps=20',
```

Reply:
Oh, nevermind then. They are both arguments to Helm and not Helmfile, passed on as `--sync-args`.

Reply:
I think it's  because it needs to be passed as an argument to the sync-args